handler/src/main/java/io/netty/handler/ssl/OpenSslSessionContext.java

```diff
@@ -97,7 +97,10 @@ public void setTicketKeys(byte[] keys) {
     }
 
     /**
-     * Sets the SSL session ticket keys of this context.
+     * Sets the SSL session ticket keys of this context. Depending on the underlying native library you may omit the
+     * argument or pass an empty array and so let the native library handle the key generation and rotating for you.
+     * If this is supported by the underlying native library should be checked in this case. For example
+     * <a href="https://boringssl.googlesource.com/boringssl/">BoringSSL</a> is known to support this.
```

Comment:
Consider referencing to the specific section: https://commondatastorage.googleapis.com/chromium-boringssl-docs/ssl.h.html#Session-tickets